chef/data_bags/crowbar/migrate/manila/002_add_manual_backend.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade(ta, td, a, d)
```

Comment:
Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.